src/Api.php

```diff
@@ -486,14 +487,12 @@ private static function getApiHost($frontendHost = false)
                 if (getenv('COMFINO_DEV_API_HOST_FRONTEND')) {
                     return getenv('COMFINO_DEV_API_HOST_FRONTEND');
                 }
-            } else {
-                if (getenv('COMFINO_DEV_API_HOST_BACKEND')) {
-                    return getenv('COMFINO_DEV_API_HOST_BACKEND');
-                }
+            } else if (getenv('COMFINO_DEV_API_HOST_BACKEND')) {
+                return getenv('COMFINO_DEV_API_HOST_BACKEND');
```

Comment:
Potrzebujemy  mieć `dev` i `prod` envy pod hosty?

Reply:
Tak, musi to być. Dzięki temu można testować notyfikacje w lokalnym środowisku i przejść cały proces łącznie z komunikacją z Sandbox. No i dzięki temu możliwe było odpalenie wtyczek na Koszuli, bez sztywnego przywiązania do Crat, chociaż ta funkcja była z myślą o lokalnej pracy robione tylko na moje potrzeby.